src/modules/esl-carousel/plugin/touch/esl-carousel.touch.mixin.ts

```diff
@@ -107,13 +113,22 @@ export class ESLCarouselTouchMixin extends ESLCarouselPlugin {
     const offset = this.getOffset(event);
 
     if (!this.$host.hasAttribute('dragging')) {
+      // Prevents draggable state if the content is scrolled
+      if (isOffsetChanged(this.startEventOffset)) return this._onPointerUp(event);
+      // Prevents draggable state if the text is selected
+      if (document.getSelection()?.isCollapsed === false) return this._onPointerUp(event);
```

Comment:
```suggestion
      if (!document.getSelection()?.isCollapsed) return this._onPointerUp(event);
```

Reply:
It have a different meaning
The undefind value should not be treated as an exite marker